apps/zbugs/src/zero-init.tsx

```diff
@@ -21,7 +22,7 @@ export function ZeroInit({children}: {children: ReactNode}) {
         }
         return login.loginState?.encoded;
       },
-    };
```

Comment:
I don't understand why this change is needed.

Reply:
What does it improve about type safety? Currently for me I do not see any type errors in my env w/o this.

The reason I'm asking is because zbugs is a testbed for how we expect Zero to be used and I wasn't expecting this to be required of users.